tests/testthat/test-renderer1-axis-angle-rotate.R

```diff
@@ -28,12 +28,22 @@ expect_rotate_anchor <- function(info, rotate, anchor){
   expect_match(rotated["style", ], paste("text-anchor:", anchor), fixed=TRUE)
   expect_match(rotated["transform", ], paste0("rotate(", rotate), fixed=TRUE)
   # http://stackoverflow.com/questions/442404/retrieve-the-position-x-y-of-an-html-element
-  tick_box <- remDr$executeScript('return document.getElementsByClassName("xaxis")[0].firstChild.getBoundingClientRect()')
-  title_box <- remDr$executeScript('return document.getElementsByClassName("xtitle")[0].getBoundingClientRect()')
-  expect_true(title_box$top >= tick_box$bottom)
+  
+  
+  tick_box_script <- 'document.getElementsByClassName("xaxis")[0].getBoundingClientRect().bottom;'
+  title_box_script <- 'document.getElementsByClassName("xtitle")[0].getBoundingClientRect().top;'
+    
+  # Evaluate scripts and get the values
+  tick_box <- remDr$Runtime$evaluate(tick_box_script, returnByValue = TRUE)$result$value
+  title_box <- remDr$Runtime$evaluate(title_box_script, returnByValue = TRUE)$result$value
+  #An offset of 2.5 px is added in line below but it may need to changed slightly if browser environment is different.
+  #more information of getBoundingClientRect() and why value it returns depends on specific browser environment is in the links below
+  #https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect
+  #https://stackoverflow.com/questions/40879171/in-javascript-why-does-getboundingclientrect-sometimes-return-floating-point
+  expect_true(title_box+2.5>= tick_box)
```

Comment:
I don't like this 2.5 is there any way to avoid that?

Reply:
use expect_gt (gt means greater than)

Reply:
also mozilla docs are no longer relevant right? (those docs are specific to firefox whereas now we only use chromote)

Reply:
> also mozilla docs are no longer relevant right? (those docs are specific to firefox whereas now we only use chromote)

 https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect#browser_compatibility, this is the only documentation I could find and chrome is there in the supported browsers in this docs. I tried to find documentation on chrome's page but could not. so I added Mozilla docs for getBoundingClientRect()

Reply:
> I don't like this 2.5 is there any way to avoid that?

@tdhock can I work on this as a separate task after you merge ? from what I have read so far I think this issue is due environment differences across browsers. I think the way chromote handles getBoundingClientRect() is slightly different from phantomjs and so the test failed only for chromote. but I think I need to do much more research to exactly figure out the underlying differences.